src/overlays/actors/ovl_Bg_Ydan_Hasi/z_bg_ydan_hasi.h

```diff
@@ -6,11 +6,16 @@
 
 struct BgYdanHasi;
 
+typedef void (*BgYdanHasiActionFunc)(struct BgYdanHasi*, struct GlobalCtx*);
+
 typedef struct BgYdanHasi {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x20];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ BgYdanHasiActionFunc actionFunc;
+    /* 0x0168 */ u8 unk_168;
+    /* 0x0169 */ s8 unk_169;
+    /* 0x016A */ s16 timer;
 } BgYdanHasi; // size = 0x016C
 
 extern const ActorInit Bg_Ydan_Hasi_InitVars;
 
-#endif
+#endif
```

Comment:
Could you re-add the empty line at the end of this header? And also use format.sh on the c file